tensorflow/contrib/lite/context.h

```diff
@@ -148,7 +148,7 @@ void TfLiteIntArrayFree(TfLiteIntArray* v);
 #define TF_LITE_ENSURE_OK(context, status) \
   do {                                     \
     if ((status) != kTfLiteOk) {           \
```

Comment:
To future proof this, should we make this:

auto status_eval = (status);
if (status_eval != kTfLiteOk) {
  return status_eval;
}